docs/documentation/fonts.md

```diff
@@ -53,6 +53,8 @@ f
 
 ## Emoji and color fonts
 
-Currently, Makie does not have the ability to draw emoji or other color fonts.
+Currently, Makie does not have the ability to draw emoji.
```

Comment:
Color fonts are actually https://www.colorfonts.wtf/

![](https://uploads-ssl.webflow.com/5a02f005838e2f000134536e/5a05c54c12148e000158ccef_ColorIsTheNewBlack-p-1080.jpeg)

that aren't rendered because of the way we render text in Makie.

I guess that colored text would be the correct name for what we're thinking of...

Reply:
Oh interesting I didn't know that I assumed it was synonymous with coloured text. IMO it would still be useful to have a pointer to the ability to change text colour since I imagine that's what most people would looking for in this context.

Reply:
Oh definitely - we'll just have to disambiguate the terms somewhere.

Reply:
```suggestion
Currently, Makie does not have the ability to draw emoji or other color fonts.  

!!! note
    Color fonts refer specifically to fonts with color embedded.  Makie does support arbitrarily coloring text using the `rich` text API.

Reply:
Maybe it would be better to be more direct and write that Makie doesn't support multi-color characters with emojis and colorfonts being examples?

Reply:
Updated to link that site as well as coloured text support.